src/platform/pdf/templates/blue_button_report.js

```diff
@@ -0,0 +1,470 @@
+            'Some records in this report are available 36 hours after providers enter them. This includes care summaries and notes, health condition records, and most lab and test results.',
+            "This report doesn't include information you entered yourself. To find information you entered yourself, go back to the previous version of Blue Button on the My HealtheVet website.",
+          ],
+          {
+            lineGap: 2,
+            paragraphGap: 10,
+            listType: 'bullet',
+            bulletRadius: 2,
+            bulletIndent: 20,
+            x: 6,
+          },
+        );
+    }),
+  );
+
+  doc.moveDown();
+  infoSection.end();
+};
+
+const generateHelpSection = (doc, parent) => {
+  const infoSection = doc.struct('Sect', {
+    title: 'Information',
+  });
+  infoSection.add(
+    createHeading(doc, 'H2', config, 'Need help?', { x: 20, paragraphGap: 12 }),
+  );
+  parent.add(infoSection);
+  infoSection.add(
+    doc.struct('List', () => {
+      doc
+        .font(config.text.font)
+        .fontSize(config.text.size)
+        .list(
+          [
+            'If you have questions about this report or you need to add information to your records, send a secure message to your care team.',
+            "If you're ever in crisis and need to talk with someone right away, call the Veterans Crisis line at 988. Then select 1.",
+          ],
+          {
+            lineGap: 2,
+            paragraphGap: 10,
+            listType: 'bullet',
+            bulletRadius: 2,
+            bulletIndent: 20,
+            x: 6,
+          },
+        );
+    }),
+  );
+
+  doc.moveDown();
+  infoSection.end();
+};
+
+const generateCoverPage = async (doc, parent, data) => {
+  await generateTitleSection(doc, parent, data);
+  await generateInfoSection(doc, parent);
+  await generateHelpSection(doc, parent);
+};
+
+const validate = data => {
+  const requiredFields = [];
+
+  const missingFields = requiredFields.filter(field => !data[field]);
+  if (missingFields.length) {
+    throw new MissingFieldsException(missingFields);
+  }
+};
+
+const generateTocItem = (doc, parent, data, pageData) => {
+  const leftMargin = 100;
+  const pages =
+    pageData.startPage === pageData.endPage
+      ? `page ${pageData.startPage}`
+      : `pages ${pageData.startPage} - ${pageData.endPage}`;
+  parent.add(
+    doc.struct('P', () => {
+      doc
+        .font(config.tocHeading.font)
+        .fontSize(config.tocHeading.size)
+        .text(`${data.title} ${pages}`, leftMargin, doc.y, {
+          lineGap: 6,
+        });
+    }),
+  );
+  parent.add(
+    doc.struct('P', () => {
+      doc
+        .font(config.text.font)
+        .fontSize(config.text.size)
+        .text(data.subtitle, leftMargin, doc.y, {
+          lineGap: 6,
+          width: 410,
+        });
+    }),
+  );
+  doc.moveDown();
+};
+
+const generateTableOfContents = (doc, parent, data, tocPageData) => {
```

Comment:
OOC what happens if the ToC ends up being longer than a single page?

Reply:
Also, great idea for testing. I'll see if the manual testers have anything like this.

Reply:
One thing I'll add is that this is a first draft. The UCD team has told me that the mockups I'm using are old and there will be an update coming soon so we will have changes and opportunities to make enhancements.

Reply:
> Thanks for bringing this up! I hadn't considered it. I'm trying to imagine how this might happen, although perhaps it should be addressed just in case. I'll look into it.

right, I'm not familiar with all possibilities of what the ToC could look like, so it may not be a concern. Just want to be sure we consider it in case more domains are added or whatever.

Reply:
After doing some research, I didn't find a good way to programmatically add extra pages to the table of contents since it is generated within the doc after the rest of the doc is generated. I would say that if the toc contents changes so that it needs more than one page, the template will have to be altered as well to accommodate the longer TOC.

Reply:
Perhaps adding a TODO comment noting the potential need and solution would suffice for this PR?

Reply:
I'm good with a TODO if the list of possible ToC items is limited to what we see above?

Reply:
Yes, it currently is limited to the items you see in the screenshot. I'm meeting with UCD in an hour and will ask them what the chances of additional items being added are or if they see any other reasons the TOC might grow.

Reply:
sounds good, thx!

Reply:
I met with Amelia Hacking today and asked how likely it is that the ToC will ever be larger than it is now. She said because the categories for medical records are fixed, she can't see an instance where the ToC would have more items than the current categories. She also said the descriptions shouldn't ever be long enough to push the current items beyond a single page. 

I added the TODO anyway, just in case this code is ever copied or used for anything else.